### DISCUSSION TITLE
Why does `chmod +x` not working on my computer

### DISCUSSION THREAD

**Discussion by JL03-Yue** [kubernetes-sigs/kubebuilder#3809] [Support]
<img width="393" alt="image" src="https://github.com/kubernetes-sigs/kubebuilder/assets/59816815/a0cb4fee-9d5f-497f-b89f-2ce37b918fdc">
After running `chmod + x`, the file still does not have execution permission. How to resolve that?

**Comment by camilamacedo86** [comment]
It seems that you are using Windows. We do not officially support Windows. 
See: [Kubebuilder Supportability](https://github.com/kubernetes-sigs/kubebuilder?tab=readme-ov-file#supportability)

However, I know that many people are able to use the tool in Windows environments. Example:
- [Issue #2940](https://github.com/kubernetes-sigs/kubebuilder/issues/2940#issuecomment-1994356354)

Also, you might be able to use [Cygwin](https://www.cygwin.com/).

Regarding your specific question about why `chmod +x` is not working on your computer, it does not seem to be related to Kubebuilder at all. It is more related to your environment, how to execute Linux commands in Windows, etc. Therefore, I hope the information provided helps you out, and please understand that we are closing this issue as resolved.
